src/actions/index.js

```diff
@@ -0,0 +1,51 @@
+import {
+  FETCH_TRENDING_GIPHYS,
+  FETCH_GIPHYS_SEARCH,
+  TOGGLE_FAVORITE_GIPHY,
+  UPDATE_FAVORITES_FILTER,
+  FETCH_GIPHYS_BY_IDS,
+  FETCH_GIPHY_DETAIL
+} from './action_types';
+
+export const fetchTrendingGiphys = () => {
+  return {
+    type: FETCH_TRENDING_GIPHYS,
+  };
+}
+
+export const fetchGiphysSearch = (term) => {
+  return {
+    type: FETCH_GIPHYS_SEARCH,
+    payload: {
+      term
+    }
+  }
+}
+
+export const toggleFavoriteGiphy = (giphyId) => {
+  return {
+    type: TOGGLE_FAVORITE_GIPHY,
+    payload: giphyId
```

Comment:
It is a good practice to wrap the payloads with an object (like you did in `fetchGiphysSearch `).